CHANGELOG.md

```diff
@@ -284,7 +284,7 @@ To decouple the `Wallet` from the `Blockchain` we've made major changes:
 ### Wallet
 - Added an option that must be explicitly enabled to allow signing using non-`SIGHASH_ALL` sighashes (#350)
 #### Changed
-`get_address` now returns an `AddressInfo` struct that includes the index and derefs to `Address`.
+`address` now returns an `AddressInfo` struct that includes the index and derefs to `Address`.
```

Comment:
You should roll back all the changes in this file since it's a historic record of what happened in prior releases and shouldn't be changed.